strawberryfields/gbs/similarity.py

```diff
@@ -400,6 +413,7 @@ def prob_event_mc(
         max_count_per_mode (int): maximum number of photons per mode in the event
         n_mean (float): total mean photon number of the GBS device
         samples (int): number of samples used in the Monte Carlo estimation
+        loss (float): fraction of photons lost in GBS
 

```

Comment:
Same here, function probably needs to raise an error if the loss is not between 0 and 1